frontend/src/components/authentication/AuthenticationTabs.jsx

```diff
@@ -28,7 +31,7 @@ class AuthenticationTabs extends Component {
     return (
       <div>
         <Container>
-          <Row className="justify-content-md-center">
```

Comment:
If this is one of our classes, and not used elsewhere, then we should probably delete it from the css too, no?

Reply:
Nix that... this is from aurora....